python/core/symbology-ng/qgssymbollayerutils.sip

```diff
@@ -501,4 +501,20 @@ class QgsSymbolLayerUtils
       */
     static void mergeScaleDependencies( int mScaleMinDenom, int mScaleMaxDenom, QgsStringMap& props );
 
+    /**
+     * Encodes a reference to a parametric SVG into SLD, as a succession of parametric SVG using URL parameters,
+     * a fallback SVG without parameters, and a final fallback as a mark with the right colors and outline for systems
+     * that cannot do SVG at all
+     * @note added in 3.0
+     */
+    static void parametricSvgToSld( QDomDocument &doc, QDomElement &graphicElem,
```

Comment:
@nyalldawson what do you think? I can still change that to break the existing pattern and build the element instead of accepting it... it would just look odd in the context where it's used, not the end of the world I presume ;-)